### DISCUSSION TITLE
diffusers 0.17.0 is out

### DISCUSSION THREAD

**Discussion by ghost** [vladmandic/sdnext#1389] [General]
https://github.com/huggingface/diffusers/releases/tag/v0.17.0

I only learnt about diffusers (and that it is a separate thing) recently. Would diffusers with the new torch compile and recent optimizations be faster than what SD.Next runs on right now? Is it difficult to integrate diffusers?

Zero shot inpainting, Controlnet support and UniDiffuser sound very interesting

**Comment by vladmandic** [comment]
diffuser integration project is in-progress, that's pretty much all i can say right now.
